perma_web/frontend/components/CreateLink.vue

```diff
@@ -127,15 +127,19 @@ const handleCaptureStatus = async (guid) => {
         const job = await response.json()
 
         return {
-            step_count: job.step_count,
+            step_count: job.step_count ?? 1,
```

Comment:
This might be overkill — happy to remove

Reply:
Yeah, get the instinct to add a fallback, but think yeah, it might be better without the it, since having a fallback implies that sometimes the API might return a capture job without a step_count attribute, which I think is never the case.

Reply:
For sure — removed this in [43fad93](https://github.com/harvard-lil/perma/pull/3581/commits/43fad932e741df891780693ee0e2221c34235834)